### ISSUE TITLE
Uncaught Error: ENOENT

### ISSUE THREAD

**Issue by nielsgoossens** [joyent/smartos-live#430]
After destroying a KVM image, something goes wrong with vmadm. If I destroy another image directly after the first, I receive the following error:

Uncaught Error: ENOENT, no such file or directory
FROM
Function.startup.resolveArgv0 (node.js:811:23)
startup (node.js:58:13)
node.js:902:3
Abort (core dumped)

Core: https://www.dropbox.com/s/lsmtyd2dn014gpr/core.vmadm.2275?dl=0

**Comment by nielsgoossens** [comment]
Just to add to this, I was logged in to the headnode over ssh when this occurred. Logging out and back in solves the problem.

**Comment by joshwilsdon** [comment]
Was your PWD was in the zoneroot of the VM you were destroying when this occurred?

**Comment by nielsgoossens** [comment]
As far as I remember, I was in /opt/. Doing a few tests now to see if I can still reproduce it..

**Comment by nielsgoossens** [comment]
No, I cannot reproduce this anymore from either /opt or the root of the zone I am trying to destroy.. If I am in the zone directory and destroy it, it will remove the contents of the zone but leave the folder itself visible. Trying to delete that gives a "Device busy".

**Comment by naterator** [comment]
I just encountered this issue and was in the zone root of the VM I was destroying as @joshwilsdon mentioned:

```
Uncaught Error: ENOENT, no such file or directory

FROM
Function.startup.resolveArgv0 (node.js:811:23)
startup (node.js:58:13)
node.js:902:3
Abort (core dumped)
```

I moved to `/root` and `vmadm` began to work as expected.

**Comment by j4s0nmchr1st0s** [comment]
Where can the beta be found?
